projects/canopy/src/lib/shadow/docs/shadow.stories.ts

```diff
@@ -27,21 +27,36 @@ export default {
 } as Meta;
 
 const template = `
+<!-- lgShadow without hover state -->
 <lg-card lgShadow>
   <lg-card-content>
     Look, I have a shadow
   </lg-card-content>
 </lg-card>
+
+<!-- lgShadow with hover state -->
+<lg-card lgShadow [hasHoverState]="hasHoverState">
```

Comment:
Should I change this to `<lg-card lgShadow [hasHoverState]="true">` since it's not likely the user will decide if they want the hover state programatically. What do you think @elenagarrone, @Xinchro?

Reply:
This is for the stories, so that a user can use the controls in Storybook to enable it